index.md

```diff
@@ -29,6 +29,8 @@ Pulp is free and open-source, and we invite you to [join us on GitHub](https://g
         <ul id="myTab" class="nav nav-tabs nav-justified">
             <li class="active"><a href="#rpm" data-toggle="tab"><i class="fa fa-cube"></i> RPM</a>
             </li>
+            <li class=""><a href="#deb" data-toggle="tab"><i class="fas fa-stop-circle"></i> Debian</a>
```

Comment:
I do not know why (docu says otherwise), but it seems to be `fa` not `fas`.
Maybe, the assets are outdated.